sysutils/dvtm/Portfile

```
@@ -18,23 +19,31 @@ long_description        dvtm brings the concept of tiling window management, \
                         it easy to work with multiple console based programs \
                         like vim, mutt, cmus or irssi.
 
-homepage                http://brain-dump.org/projects/dvtm/
-master_sites            ${homepage}
+homepage                https://brain-dump.org/projects/dvtm
+master_sites            ${homepage}/
```

We should use the canonical homepage -- the one the web server redirects to. See https://trac.macports.org/wiki/howto/AvoidRedirects
```suggestion
homepage                https://www.brain-dump.org/projects/dvtm/
master_sites            ${homepage}
```